### DISCUSSION TITLE
Publish a message to the stream, why the stream has two message records

### DISCUSSION THREAD

**Discussion by MacGyverXie** [nats-io/k8s#471] [General]
![image](https://user-images.githubusercontent.com/100560322/157843470-ee737d9a-fb4b-4612-9268-dc8849afbe75.png)

**Comment by wallyqs** [comment]
Is this with latest version of the server?

**Reply by MacGyverXie** [reply]
I'm sorry, this should be caused by duplicate topics in the stream I created
